### PR TITLE
chore(journeys): implement better saucelabs practices

### PR DESCRIPTION
**Pull request by moriahmaney**

### CODE REVIEW

**Comment by adamweeks** [comment]
Deploy preview for *hopeful-hamilton-4090b9* ready!

Built with commit 67dcfcbd582bb545eb171ee9a6517b1905f51b7d

https://deploy-preview-747--hopeful-hamilton-4090b9.netlify.com

File: test/journeys/specs/space/global/messaging.js [hunk 1]
```diff
@@ -19,6 +20,9 @@ import {
 describe('Widget Space', () => {
   const browserLocal = browser.select('browserLocal');
   const browserRemote = browser.select('browserRemote');
+  const jobName = 'react-widget-space-global';
```
Comment (adamweeks) [inline test/journeys/specs/space/global/messaging.js]: Almost feel like these job names should be in a shared file somewhere so we don't get strings mucked up along the way.

File: Jenkinsfile [hunk 2]
```diff
@@ -119,17 +119,22 @@ ansiColor('xterm') {
...
-             wait
-             '''
-             archiveArtifacts 'reports/**/*'
-             junit '**/reports/junit/wdio/*.xml'
+              sh '''#!/bin/bash -ex
+              source ~/.nvm/nvm.sh
+              nvm use 8.9.1
+              NODE_ENV=test npm run build:package widget-space && npm run build:package widget-recents
+              // set -m sets all integration commands under the same job process
+              // || kill 0 after each integration command will kill all other jobs in the parent process if the integration command preceding it fails with a non-zero exit code
```
Comment (adamweeks) [inline Jenkinsfile]: Thanks for the comments!

File: test/journeys/specs/space/global/messaging.js (hunk 1 above)
Comment (moriahmaney) [reply test/journeys/specs/space/global/messaging.js]: Sure, done!

**Pull request merged**